py-polars/src/dataframe.rs

```diff
@@ -1222,23 +1222,35 @@ impl PyDataFrame {
     }
 
     #[cfg(feature = "pivot")]
+    #[pyo3(signature = (index, columns, values, maintain_order, sort_columns, aggregate_expr, separator))]
     pub fn pivot_expr(
         &self,
-        values: Vec<String>,
         index: Vec<String>,
         columns: Vec<String>,
+        values: Option<Vec<String>>,
         maintain_order: bool,
         sort_columns: bool,
         aggregate_expr: Option<PyExpr>,
         separator: Option<&str>,
     ) -> PyResult<Self> {
         let fun = if maintain_order { pivot_stable } else { pivot };
         let agg_expr = aggregate_expr.map(|expr| expr.inner);
+        let values = match values {
+            Some(v) => v,
+            None => {
+                // No value columns provided, use remaining columns
+                self.columns()
+                    .iter()
+                    .map(|s| s.to_string())
+                    .filter(|s| !(index.contains(s) | columns.contains(s)))
+                    .collect()
+            },
+        };
```

Comment:
This should be implemented on the Rust side.

Reply:
I'm confused, this is on the rust side, is it not?

Reply:
Ahh, you mean in the polars crate. Got it.

Reply:
Yes. That way Rust users can also take advantage of this improvement!